packages/aws-cdk-lib/aws-s3/lib/bucket.ts

```diff
@@ -1400,6 +1400,145 @@ export abstract class TargetObjectKeyFormat {
   public abstract _render(): CfnBucket.LoggingConfigurationProperty['targetObjectKeyFormat'];
 }
 
+/**
+ * Specifies which Amazon S3 objects to replicate and where to store the replicas.
+ */
+export interface ReplicationRule {
+  /**
+   * A container for information about the replication destination.
+   */
+  readonly destination: ReplicationDestination;
+
+  /**
+   * Specifying S3 Replication Time Control (S3 RTC),
+   * including whether S3 RTC is enabled and the time when all objects and operations on objects must be replicated.
+   *
+   * @default - S3 Replication Time Control is not enabled
+   */
+  readonly replicationTimeControl?: boolean;
+
+  /**
+   * A container specifying replication metrics-related settings enabling replication metrics and events.
+   *
+   * @default - Replication metrics are not enabled
+   */
+  readonly metrics?: boolean;
```

Comment:
above interface eg. `IReplicationTimeValue` should help here as well in case we have a modifiable value for minutes in the future